dlna/dms/cds.go

```diff
@@ -588,22 +584,19 @@ func (me *contentDirectoryService) objectHasChildren(
 
 // Returns the actual local filesystem path for the object.
 func (o *object) FilePath() string {
-	return filepath.Join(o.RootObjectPath, filepath.FromSlash(o.Path))
+	return path.Join(o.RootObjectPath, path.Clean(o.Path))
 }
 
 // Returns the ObjectID for the object. This is used in various ContentDirectory actions.
 func (o object) ID() string {
-	if !path.IsAbs(o.Path) {
-		log.Panicf("Relative object path: %s", o.Path)
-	}
 	if len(o.Path) == 1 {
 		return "0"
 	}
 	return url.QueryEscape(o.Path)
 }
 
 func (o *object) IsRoot() bool {
-	return o.Path == "/"
+	return o.Path == "."
```

Comment:
Are you sure about changing this part? Same with the other usage. Can you make `/` rooted in whatever subtree fs.FS is?

Reply:
Ah yeah, the interface I was going for was to expose `/` as the root of the FS exposed to the device. I understand `.` would be used on the server side that makes sense.

Reply:
I guess what I was going for was retaining the `/` being the root of the path as seen from the client. That has the advantage that if the client switches to using `/` instead of `.`, we can be more certain the server won't escape the subtree that was intended to be shared. Also I need to check that everything still works in a quick test.